packages/cli/src/util/input/select-org.ts

```diff
@@ -26,10 +26,14 @@ export default async function selectOrg(
   }
 
   const choices: Choice[] = [
-    {
-      name: user.name || user.username,
-      value: { type: 'user', id: user.id, slug: user.username },
-    },
+    ...(user.version === 'northstar'
+      ? []
+      : [
+          {
+            name: user.name || user.username,
+            value: { type: 'user', id: user.id, slug: user.username },
+          } as const,
+        ]),
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏extract into variable, or something

I find this complex set of expressions hard to read. Can we extract some named values or otherwise make this clearer?

Reply:
Fixed by 761ff070f